forge-core/src/main/java/forge/card/CardRules.java

```diff
@@ -485,6 +497,7 @@ public final CardRules getCard() {
             result.setNormalizedName(this.normalizedName);
             result.meldWith = this.meldWith;
             result.partnerWith = this.partnerWith;
+            result.addsWildCardColor = this.addsWildCardColor;
```

Comment:
there's no point for changing the Reader in your current code?

Reply:
From my understanding, this represents the in-game display of the card.  In this update, I did not make it so that these cards would reflect their proper color in game, but I felt it would be best to make the modification that would probably be needed for future development at the same time.